content/millennium/dstu2/scheduling/schedule.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Schedule | DSTU 2 API
+---
+
+# Schedule
+
+* TOC
+{:toc}
+
+## Overview
+
+Schedule resources provide a container for time-slots that can be booked using an appointment. It provides the window of time (period) that slots are defined for and what type of appointments can be booked.
+<br/><br/>
+The following fields are returned if valued:
+
+* [Type](http://hl7.org/fhir/DSTU2/schedule-definitions.html#Schedule.type){:target="_blank"}
+* [Actor](http://hl7.org/fhir/DSTU2/schedule-definitions.html#Schedule.actor){:target="_blank"}
+* [Location Extension](#extensions)
+
+## Terminology Bindings
+
+<%= terminology_table(:schedule, :dstu2) %>
+
+## Extensions
+
+* [Location]
+
+### Custom Extensions
+
+All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/dstu2/StructureDefinition/{id}`
+
+ID             | Value\[x] Type    | Description
+---------------|-------------------|----------------------------------------------------------------------------------
+`location`      | [`Reference`]          | Reference to the location corresponding to this schedule, where the appointment can be booked.
+
+## Search
+
+Search for Schedules that meet supplied query parameters:
+
+    GET /Schedule?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true) %>
+
+### Parameters
+
+ Name         | Required?                             | Type          | Description
+--------------|---------------------------------------|---------------|------------------------------------------------------------------------------------
+`_id`         | Y                                     | [`token`]     | The logical resource id associated with the resource.
+
+### Headers
+
+ <%= headers %>
+
+### Example
```

Comment:
Do we happen to have an example where the type is coded in sandbox?

Reply:
<bump> on this question :)

Reply:
Currently I am not getting valid responses for these resources since the toggles are disabled.
I have logged jiras for Schedule\Slot search and read, and Appointment create.
Once those are enabled, I'll update the examples with valid requests and responses.

Reply:
Addressed in https://github.com/cerner/fhir.cerner.com/pull/153/commits/5e9098c298c1764c1966b824fc93c33896719b31